src/libcrun/linux.c

```diff
@@ -1611,7 +1611,12 @@ libcrun_create_dev (libcrun_container_t *container, int devfd, int srcfd,
   int rootfsfd = get_private_data (container)->rootfsfd;
   const char *rootfs = get_private_data (container)->rootfs;
   size_t rootfs_len = get_private_data (container)->rootfs_len;
-  const char *rel_dev = relative_path_under_dev (device->path);
+  // Normalize the path by removing trailing slashes.
+  cleanup_free char *normalized_path = xstrdup (fullname);
+  consume_trailing_slashes (normalized_path);
+  if (normalized_path[0] == '\0')
```

Comment:
```suggestion
  if (is_empty_string(normalized_path))
```

This can also be used to check if the `fullname` variable is empty, per @giuseppe's request.

Reply:
@sohankunkerkar, not using `is_empty_string()` here?

Reply:
The helper would be fine. I think we should use it. Also, it improves readability a bit (since the intent will be obvious given the function name).

Reply:
A bit of a shame we didn't use the helper here. Consistency would be nice.